apps/src/code-studio/pd/professional_learning_landing/plcElements/EnrollmentUnitAssignment.jsx

```diff
@@ -1,6 +1,6 @@
 import React, {PropTypes} from 'react';
 import color from '../../../../util/color';
-import ModuleAssignment from './moduleAssignment';
+import ModuleAssignment from './ModuleAssignment';
```

Comment:
yay! The lowercase and snake_case components and files in PLC land bug me more than they should 🙂 - this is a great start to making them more consistent with the rest of our component names.